2.1.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <meta charset="UTF-8" ; />
+        <body>
+                <p>The highest number is <span id="demo"></span>.</p>
+            <script>
+                var num = [110, 220, 3030, 4040, 500500];
+                document.getElementById("demo").innerHTML = myArrayMax(num);
+            function myArrayMax(arr){
+                let len = arr.length
+                let max= - Infinity;
+                while (len--){
```

Comment:
#challenge
Try to use for instead of while because we know how many iterations needs to be done.

Reply:
Done and sent